packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -486,6 +489,113 @@ $Calendar-dayPicker-margin: 10px;
     }
   }
 
+  // Show the month picker over date picker (1 column layout), if enabled
+  .pickerIsMonthPickerOverlayVisible {
+    // Show the month picker.
+
+    .holder {
+      width: 230px;
+    }
+
+    .monthPicker {
+      display: block;
+    }
+    // Style the the month and year pickers.
+    .monthPicker,
+    .yearPicker {
+      position: relative;
+    }
+    // Size and position of the month and year labels.
+    .month {
+      @include margin-left(12px);
+    }
+
+    .optionGrid {
+      width: 220px;
+      @include margin(10px, 0, 0, 0px);
+    }
+    // This component is only used on small displays.
```

Comment:
I will be consolidating (more details in one of the messages below).  I kept all my new css separate, but it looks like that wasn't necessary.